src/main/java/net/dv8tion/jda/api/events/guild/member/GuildMemberUpdateNicknameEvent.java

```diff
@@ -24,14 +24,14 @@
  *
  * <p>Can be used to retrieve members who change their nickname, triggering guild, the old nick and the new nick.
  */
-public class GuildMemberNickChangeEvent extends GenericGuildMemberEvent
+public class GuildMemberUpdateNicknameEvent extends GenericGuildMemberEvent
```

Comment:
This should implement `UpdateEvent` as well then

Reply:
Should `getPropertyIdentifier()` return `nick` or `nickname`?

Reply:
Following the api docs, it should be `nick`